sjb/generate.py

```diff
@@ -99,11 +99,40 @@ def load_configuration(config_path):
 
     # next, repositories will be synced to the remote VM
     sync_actions = []
+    repo_names=[repository["name"] for repository in job_config.get("sync_repos", [])]
+
+    # determine if test is a pull request
+    is_pull_request = False
+    for repository in job_config.get("sync_repos", []):
+        if repository.get("type"):
+            is_pull_request = True
+
+    # Determine the name of dependency repository that will serve as prefix to 'XXX_TARGET_BRANCH'
+    # variable, so we can set the matching branch for input repository.
+    # So far the repository that are dependent to each other are 'origin' and 'openshift-ansible'.
```

Comment:
Comment is out of date already -- remove it, rest of code is easy enough to follow